src/masternodes/mn_rpc.h

```diff
@@ -61,5 +61,6 @@ CAccounts SelectAccountsByTargetBalances(const CAccounts& accounts, const CBalan
 void execTestTx(const CTransaction& tx, uint32_t height, CTransactionRef optAuthTx = {});
 CScript CreateScriptForHTLC(const JSONRPCRequest& request, uint32_t &blocks, std::vector<unsigned char>& image);
 CPubKey PublickeyFromString(const std::string &pubkey);
+std::optional<std::pair<CAmount, CAmount>> GetFuturesBlockAndReward();
```

Comment:
general question: why call it "reward" everywhere? isn't it a "offset", "fee" or something like that? definitly not a "reward" right?

Reply:
This is correct. A bit of obsolete terminology that slipped through and got shipped. Internal code can be easily refactored. But change on attrs is probably now have to be there for a bit.

Reply:
Resolving this at the moment, since there's no action points unfortunately - and was a bit constrained on bandwidth to correct this in time.

Reply:
thx. could I do anything different on reporting the next time? so that it gets attention before getting shipped?